### ISSUE TITLE
ionic emulate livereload not exposed on localhost

### ISSUE THREAD

**Issue by jasperkuperus** [driftyco/ionic-cli#427]
We used to start the simular using the following command:

```
ionic emulate ios -l --address=localhost
```

One of the benefits of this was that the contents of the simulator are also available on `localhost:8100`. Updating my CLI to 1.4.3 broke this and only exposed the app on the IP address. By inspecting the code I found a hidden flag which is not documented:

```
ionic emulate ios -l --allIps
```

This works like a charm, but isn't documented anywhere. Could you please elaborate on this whether this is the correct approach? If so, could you please document this? :-)

**Comment by jbavari** [comment]
@jasperkuperus You bet. On it now.

**Comment by jbavari** [comment]
@jasperkuperus The documentation for run/emulate shows the `--all` option. 

I've updated the code to respect that flag in [this commit](https://github.com/driftyco/ionic-cli/commit/96140a6de1444435a202e7b2b0bcc777371c01f8).

Published 1.4.4 with this change. Hope this helps!

**Comment by jasperkuperus** [comment]
Ok, great, that works. Would it be an idea to also document this in the CLI help text? E.g.:

```
emulate [options] <PLATFORM> ..............  Emulate an Ionic project on a simulator or emulator
        [--livereload|-l]  ................  Live reload app dev files from the device (beta)
        [--all]  ..........................  Have the server listen on all addresses (0.0.0.0)

```
